packages/contracts/test/core/V3/prohibition/GenArt721CoreV3_Integration_PROHIBITION.test.ts

```diff
@@ -377,7 +377,7 @@ for (const coreContractName of coreContractsToTest) {
           expect(coreType).to.be.equal("GenArt721CoreV3_Engine_Flex");
         } else {
           // coreType is same for GenArt721CoreV3 & GenArt721CoreV3_Explorations
-          expect(coreType).to.be.equal("GenArt721CoreV3");
+          throw new Error("Untested core contract version");
```

Comment:
nit - should we remove the comment on line 369 - 370 (GH won't let me comment up there 😓 )

Reply:
thanks - updated on feature branch directly in [e7b73d8](https://github.com/ArtBlocks/artblocks-contracts/pull/1470/commits/e7b73d854f61a0b415c144a7d9233a98c77fc4a0)